packages/authentication/modules/user/useSimpleTokenUser/index.ts

```diff
@@ -5,13 +5,13 @@ import Cookies from 'js-cookie'
 
 import UserApi, { USER_API_KEY } from '../../../services/user'
 import { IUser } from '../../../types/user'
-import { IUseSimpleTokenUser } from './types'
+import { IUseSimpleTokenUser, IUseSimpleTokenUserResult } from './types'
 
 const useSimpleTokenUser = <TUser extends Partial<IUser>>({
   options,
   cookieName = ACCESS_COOKIE_NAME,
   ApiClass = UserApi,
-}: IUseSimpleTokenUser<TUser> = {}) => {
+}: IUseSimpleTokenUser<TUser> = {}): IUseSimpleTokenUserResult<TUser> => {
```

Comment:
do we need that? letting TS infer the return type is often easier and more accurate

Reply:
It was just for a test so I moved the type to the test and let this one be inferred